docs/functions/map_to_s3.rst

```diff
@@ -14,24 +14,22 @@ Deployment
 ----------
 
 This function may be deployed like any other AWS Lambda function using the
-following event and environment variables.
+following event and configuration file.
 
 Event
 ^^^^^
 The event for this function must be a CloudFront distribution origin-request to
 an S3 bucket.
 
-Environment Variables
-^^^^^^^^^^^^^^^^^^^^^
-- DB_TABLE_NAME
-    (Required)
+Configuration
+^^^^^^^^^^^^^
+The map_to_s3 function must be deployed with the map_to_s3.json configuration
```

Comment:
What does that mean exactly?  The file must be within the deployment .zip ?

Reply:
Yes

Reply:
No, it was just for me to understand during review.  I think this is OK - creation of deployment package obviously should be automated at some point, then I wouldn't see much point duplicating all the details in the docs.